ui/src/album/AlbumListView.js

```diff
@@ -28,6 +28,7 @@ import {
   RatingField,
 } from '../common'
 import config from '../config'
+import './Datag.css'
```

Comment:
Please use `useStyles`. The way it is implemented here the change becomes global (to **all** MuiTableCell and MuiTableRow elements in **all** views)

Reply:
@deluan, thanks for your review; I have added the new component `DataGrid` in `common`,